plugins/modules/ntnx_users_api_key_v2.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Nutanix
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+module: ntnx_users_api_key_v2
+short_description: Module to create/delete key of a requested type for a user in Nutanix Prism Central.
+version_added: 2.2.0
+description:
+    - This module can be used to create or delete a key of a requested type for a user in Nutanix Prism Central.
+    - If state is present, this module will create a key for the user.
+    - If state is absent, this module will delete the key for the user.
+    - This modules uses PC v4 APIs based SDKs.
+options:
+    user_ext_id:
+        description:
+            - The external identifier of the user.
+        type: str
+        required: true
+    ext_id:
+        description:
+            - The external identifier of the key.
+            - Required in case of deleting the key.
+        type: str
+    name:
+        description:
+            - Identifier for the key in the form of a name.
+            - Required in case of creating new key.
+        type: str
+    description:
+        description:
+            - Brief description of the key.
+        type: str
+    key_type:
+        description:
+            - The type of the key.
+            - Required in case of creating new key.
+        type: str
+        choices:
+            - API_KEY
+            - OBJECT_KEY
+    creation_type:
+        description:
+            - The creation mechanism for the key.
+        type: str
+        choices:
```

Comment:
All this choices can be configure by user ? Or anything is internal to PC. Please check once